val/include/sbsa_avs_cfg.h

```diff
@@ -35,5 +35,7 @@ extern uint64_t g_stack_pointer;
 extern uint64_t g_exception_ret_addr;
 extern uint64_t g_ret_addr;
 extern uint32_t g_curr_module;
+extern uint32_t g_single_test;
```

Comment:
Should the variables g_curr_module and g_single_test be assigned to the sentinel values?